src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/DataReaderTest/DataReaderTest.cs

```diff
@@ -225,5 +226,81 @@ private static bool IsColumnBitSet(SqlConnection con, string selectQuery, int in
             }
             return columnSetPresent;
         }
+
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup))]
+        public static void CheckHiddenColumns()
+        {
+            // hidden columns can be found by using CommandBehavious.KeyInfo or at the sql level
```

Comment:
```suggestion
            // hidden columns can be found by using CommandBehavior.KeyInfo or at the sql level
```

Reply:
Done, Though I disagree on principle with the spelling of behaviour without a u it's a reference to the type so I did it with gritted teeth.